src/Admin/AdminHelper.php

```diff
@@ -224,6 +244,8 @@ public function appendFormFieldElement(AdminInterface $admin, $subject, $element
      * @param object $object
      *
      * @throws \RuntimeException
+     *
+     * @return object
```

Comment:
Is this really needed? You pass an object by reference.

Reply:
Yes, I need to get the instance created.

Reply:
Ain't it the same instance you pass as an argument?

Reply:
No, I pass `objet`, I create a variable `instance` and I return `instance`.